configs/enet/enet_cityscapes_1024x512_adam_0.002_80k.yml

```diff
@@ -0,0 +1,42 @@
+iters: 80000
+batch_size: 8
+
+train_dataset:
+  type: Cityscapes
+  dataset_root: data/cityscapes
+  transforms:
+    - type: RandomPaddingCrop
+      crop_size: [1024, 512]
+    - type: RandomDistort
+      brightness_range: 0.4
+      contrast_range: 0.4
+      saturation_range: 0.4
+    - type: Normalize
+  mode: train
+
+val_dataset:
+  type: Cityscapes
+  dataset_root: data/cityscapes
+  transforms:
+    - type: Normalize
+  mode: val
```

Comment:
You can inherit the cityscapes.yml which have the same valset and loss config as yours. Just set the __base__ as following and you can refer to this: https://github.com/PaddlePaddle/PaddleSeg/blob/release/2.3/configs/deeplabv3/deeplabv3_resnet50_os8_cityscapes_1024x512_80k.yml
```_base_: '../_base_/cityscapes.yml'```

Reply:
When you set this, it would be great if you adjust the order of configs like that config as well.

Reply:
Done.